sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -382,11 +383,17 @@ class DataFrameReader private[sql](sparkSession: SparkSession) extends Logging {
     }
 
     verifyColumnNameOfCorruptRecord(schema, parsedOptions.columnNameOfCorruptRecord)
+    val dataSchema = StructType(schema.filterNot(_.name == parsedOptions.columnNameOfCorruptRecord))
```

Comment:
Nit: `dataSchema ` -> `actualSchema`? Be consistent what we did in the other place?